contracts/svm/testing/tests/swap.rs

```diff
@@ -23,6 +24,7 @@ use {
         },
     },
     testing::{
+        assert_all_token_balances,
```

Comment:
can you replace the other appearances of `token_balance_matches` with the macro, if not too hard